src/frontend/Sider/Behandling/Brev/Html.tsx

```diff
@@ -57,7 +54,7 @@ const HtmlBrev: React.FC<Props> = ({
                     <div className="ikon-og-dato">
                         <NavIkon />
                         {/* placeholder i tilfelle brevet besluttes annen dag enn det lages */}
-                        <p>{brevOpprettetDatoPlaceholder || brevOpprettetDato}</p>{' '}
+                        <p>{brevOpprettetDatoPlaceholder}</p>{' '}
```

Comment:
Kan man fjerne tomt mellomrom etter?
```suggestion
                        <p>{brevOpprettetDatoPlaceholder}</p>
```